spec/services/shopify_import/products/create_spec.rb

```diff
@@ -74,11 +91,11 @@
         context 'option values' do
           let(:f_option_type) { Spree::OptionType.find_by(name: shopify_product.options.first.name.downcase) }
           let(:s_option_type) { Spree::OptionType.find_by(name: shopify_product.options.first.name.downcase) }
-          let(:f_ot_values) { shopify_product.options.first.values }
-          let(:s_ot_values) { shopify_product.options.last.values }
+          let(:f_ot_values) { shopify_product.options.first.values.map(&:downcase) }
+          let(:s_ot_values) { shopify_product.options.last.values.map(&:downcase) }
 
           it 'creates option values' do
-            expect { subject.save! }.to change(Spree::OptionValue, :count).by(4)
+            expect { subject.save! }.to change(Spree::OptionValue, :count).by(6)
```

Comment:
This really proves my point from above. I, as a reader of this spec, have no idea what's going on here. The only thing I can understand is that some option values get created. Why 6? No idea. There are two option types (not six). And they are named `f_option_type`. Please write full variable name when possible. In most cases it's really not that much longer. If `f_option_type` stands for `first_option_type`, I'd vote for `option_type1`